### DISCUSSION TITLE
🐞 Alert component bug

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3519] [General] [answered]
helloooo,

Ran into a few issues with the Alert component:

1. Formatting seems to be off - title and text aren't staying within the same paragraph/paragraph text isn't staying within the component:

![Screenshot 2023-10-04 at 10 47 47 AM](https://github.com/twilio-labs/paste/assets/87141551/9455190a-fb93-4270-a80a-54008bb7e24f)

2. When switching back and forth between alert types, text color isn't changing:
![Screenshot 2023-10-04 at 10 52 49 AM](https://github.com/twilio-labs/paste/assets/87141551/0a7fd716-69d8-44ca-ae91-fb3673586ff2)

3. Neutral is the only Alert that is dismissable.

**Comment by serifluous** [comment, marked as answer]
1. That's unfortunately intentional right now to keep the title and text as separate layers (so we don't run into that text color problem in your second screenshot that that was supposed to fix :\ ). You'll have to set the paragraph to "fill" width.
2. Can you share reproduction steps, or a link to the figma file? I _think_ the component might have to pull in an update.
3. Just pushed up a fix, thanks!

**Reply by ceyerkes** [reply]
thanks @serifluous! 
1. Noted!
2. Link to [frame](https://www.figma.com/file/IY1FfId7GjRWpXUABUwF2l/%F0%9F%94%AE-Predictions-Design-%F0%9F%94%AE?type=design&node-id=4703-11154&mode=design&t=yxOiMIRjrnXOFTal-4) here.
3. Thank you!

**Reply by serifluous** [reply]
For number 2, I'm guessing it's not swapping because there's a different text color set on the "link text" in that layer, so it might be maintaining all the properties on it when it goes from Neutral -> Error. I'd recommend resetting changes on that layer, and then re-entering the text content and applying the link color style.

Figma just gets suuuper finicky when there are multiple text and color styles on the same text layer.

**Comment by ceyerkes** [comment]
gotcha - is this the intended layout now? 

![Screenshot 2023-10-04 at 2 06 26 PM](https://github.com/twilio-labs/paste/assets/87141551/086b533d-ae89-45c8-9733-60da2b609f86)

**Reply by serifluous** [reply]
It's not—just a limitation of Figma unfortunately.

I just published an update that adds the next line of text so you can cut and paste any overflow text into it.

**Reply by ceyerkes** [reply]
thanks @serifluous !!
